dsa-23au/notes/docs/week-01/2023-09-28-Co-Creation.md

```diff
@@ -2,8 +2,80 @@
+1. For the first week, I have and still am learning how to navigate github and gitpod.-Torsten
+2. Familiarization with GIT is helpful for the first two days of class to reduce the confusion one experiences.
+3. Working with other people helps solve issues better versus working alone, Especially for new people. It is much more fun to code with others, solo is made for sleepless nights.
+4. A refresher online course before class on Java would be beneficial.
+5. Collaboration is much more rewarding in person than remotely.
 what are we going to learn as a team
+
 Change makers lab is from 1pm -5pm 
 Is this classs monday as well? Yes this class is Mnday Tuesday Thursday
 Taking this Course has already Changed my life !
 I was choosing to work a job from 8am- 4pm runing netwrok data abling for $26 an hour but chose this class instead I am very very greatful
+
+Readjusting work to fit around this class was wise.
+Learning how to work Gitpod/Github is hard.
+I need to get blue light glasses to protect my eyes.
+Bringing water is a great idea so I can focus.
+I consume caffeine in order to stay awake, but it would be better to drink Matcha than Coffee.
+
+I took the Changemaker Lab in Fall 2022 and I learned lots of leaderships skills and workshopping skills. I love Destiny 2 and Sonic the Hedgehog. I cant wait to buy my new GPU so I can do more in programming and designs/modeling. I am excited to graduate this year with BS and get my cybersecurity. I am also excited to further my skills at game development.
+
+////FOR OCT 5 assignment This past week, I had to brush up on my programming and psuedocode skills that we learned in CSF. I have grown pretty comfortable with arrays and linked lists. My syntax skills have gotten better as I have been using Java API (I urge you to use that as a tool). I have also been practicing call function because that is a great skill to learn with arrays and linked lists. Also ALWAYS PUT COMMENTS IN YOUR CODE!!!
+
+<<<<<<< HEAD Paul is cryptogoth.
+
+we need more practice with git and gitpod and github.
+
+SSH keys are confusing and I just want to use my VS code not GitPods'.
+
+we are lost but so is everyone else so its okay.
+
+I still don't know what to make a list of.
+
+======= We learned that data structures are found commonly in everday life.
+
+They are used to organize data.
+
+Everyone has their own way of setting things up.
+
+Algorithms are processes that take an input state and put out an output state.
+
+They will always have an invariant, which is a known stating state.
+ Readjusting work to fit around this class was wise.
+Learning how to work Gitpod/Github is hard.
+I need to get blue light glasses to protect my eyes.
+Bringing water is a great idea so I can focus.
+I consume caffeine in order to stay awake, but it would be better to drink Matcha than Coffee.
+Git is alot of fun
+
+1. I would like more direction from the instructer. I find this horribly challenging not having any steps to guide us. 
```

Comment:
Im glad to see in the co-creation sentences that we are all in the same boat for the most part in terms of confusion.